core/snippets/snippets_insert_raw_text.py

```diff
@@ -27,9 +42,14 @@ def insert_snippet_raw_text(body: str):
         right(stop.col)
 
 
+def compute_indentation_as_spaces():
+    return " " * settings.get("user.snippets_raw_text_spaces_per_tab")
+
+
 def parse_snippet(body: str):
     # Some IM services will send the message on a tab
-    body = re.sub(r"\t", INDENTATION, body)
+    if settings.get("user.snippets_raw_text_convert_indentation"):
+        body = re.sub(r"\t", compute_indentation_as_spaces(), body)
```

Comment:
Have you verified that if you don't replace tabs with spaces the calculation further down of columns still work?